lib/ddtrace/propagation/distributed_headers/b3.rb

```diff
@@ -0,0 +1,46 @@
+require 'ddtrace/ext/distributed'
+require 'ddtrace/propagation/distributed_headers/base'
+require 'ddtrace/propagation/distributed_headers/headers'
+
+module Datadog
+  module DistributedHeaders
+    # B3 provides helpers to inject or extract headers for B3 style headers
+    class B3
```

Comment:
If this component is never intended to be initialized or hold state, then it should probably be converted to a `Module`.

Reply:
This might be carry over when I was having trouble getting `extend Base` to work, so can move back to a `module`